src/globus_cli/commands/timer/create.py

```diff
@@ -0,0 +1,288 @@
+    required=True,
+    type=str,
+    help="ID for the destination transfer endpoint",
+)
+@click.option(
+    "--label",
+    required=False,
+    type=str,
+    help=(
+        "An optional label for the transfer operation, up to 128 characters long. Must"
+        " contain only letters/numbers/spaces, and the following characters: - _ ,"
+    ),
+)
+@click.option(
+    "--stop-after-date",
+    required=False,
+    type=click.DateTime(formats=DATETIME_FORMATS),
+    help=("Stop running the transfer after this date"),
+)
+@click.option(
+    "--stop-after-runs",
+    required=False,
+    type=int,
+    help=("Stop running the transfer after this number of runs have happened"),
+)
+@click.option(
+    "--sync-level",
+    required=False,
+    type=int,
+    help=(
+        "Specify that only new or modified files should be transferred. The behavior"
+        " depends on the value of this parameter, which must be a value 0–3, as"
+        " defined in the transfer API: 0. Copy files that do not exist at the"
+        " destination. 1. Copy files if the size of the destination does not match the"
+        " size of the source. 2. Copy files if the timestamp of the destination is"
+        " older than the timestamp of the source. 3. Copy files if checksums of the"
+        " source and destination do not match."
+    ),
+)
+@click.option(
+    "--encrypt-data",
+    is_flag=True,
+    default=False,
+    help="Whether Transfer should encrypt data sent through the network using TLS",
+)
+@click.option(
+    "--verify-checksum",
+    is_flag=True,
+    default=False,
+    help=(
+        "Whether Transfer should verify file checksums and retry if the source and"
+        " destination don't match"
+    ),
+)
+@click.option(
+    "--preserve-timestamp",
+    is_flag=True,
+    default=False,
+    help=(
+        "Whether Transfer should set file timestamps on the destination to match the"
+        " origin"
+    ),
+)
+@click.option(
+    "--item",
+    "-i",
+    required=False,
+    type=(str, str, bool),
+    multiple=True,
+    help=(
+        "Used to specify the transfer items; provide as many of this option as files"
+        " to transfer. The format for this option is `--item SRC DST RECURSIVE`, where"
+        " RECURSIVE specifies, if this item is a directory, to transfer the entire"
+        " directory. For example: `--item ~/file1.txt ~/new_file1.txt false`"
+    ),
+)
+@click.option(
+    "--skip-source-errors",
+    is_flag=True,
+    default=False,
+    help=(
+        "Skip files or directories on the source endpoint that hit PERMISSION_DENIED or"
+        " FILE_NOT_FOUND errors."
+    ),
+)
+@click.option(
+    "--fail-on-quota-errors",
+    is_flag=True,
+    default=False,
+    help="Cancel the submitted transfer tasks if QUOTA_EXCEEDED errors are hit.",
+)
+@click.option(
+    "--recursive-symlinks",
+    required=False,
+    type=click.Choice(["ignore", "keep", "copy"], case_sensitive=False),
+    default="ignore",
+)
```

Comment:
Although this option is supported by Transfer, it doesn't do anything (except on a very small number of special endpoints). We should therefore not expose it here -- it has caused confusion in the past, so the docs now have a special note about it.

(And yes, we might have it in other places where it needs to be removed.)

Reply:
Excellent!